railties/test/code_statistics_test.rb

```diff
@@ -17,4 +17,18 @@ def teardown
       @code_statistics = CodeStatistics.new(['tmp dir', @tmp_path])
     end
   end
+
+  test 'ignores hidden files' do
+    content = <<-'CODE'
+      def foo
+        puts 'foo'
+      end
+    CODE
+
+    path = File.join(@tmp_path, '.example.rb')
+    File.write path, content
+    stats = CodeStatistics.new(['hidden file', @tmp_path])
+
+    assert_equal 0, stats.instance_variable_get("@statistics")['hidden file'].lines
```

Comment:
No worries :smile:

You're trying to remove a raised exception. I find `assert_nothing_raised` to better capture that than reaching into the statistics internal state 😁